docs/quickstart.md

```diff
@@ -13,32 +13,463 @@ You can find the latest Sedna release [here](https://github.com/kubeedge/sedna/r
+        ...
+    
+    def evaluate(self, data, **kwargs):
+       ...
+
+    def predict(self, data, **kwargs):
+        ...
+
+    def load(self, model_url, **kwargs):
+        ...
+
+    def save(self, model_path, **kwargs):
+        ...
+
+    def get_weights(self):
+        ...
+
+    def set_weights(self, weights):
+        ...
+```
+
+##### 2.3.2 Dataset prepare
+
+In incremental_learning jobs, the following files will be indispensable:
+
+- base model: tensorflow object detection Fine-tuning a model from an existing checkpoint.
+- deploy model:  tensorflow object detection model, for inference.
+- train data: images with label use for Fine-tuning model.
+- test data: video stream use for model inference.
+
+```bash
+# download models, including base model and deploy model.
+
+cd /
+wget https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/models.tar.gz
+tar -zxvf models.tar.gz
+
+# download train data
+
+cd /data/helmet_detection  # notes: files here will be monitored and used to trigger the incremental training
+wget  https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/dataset.tar.gz
+tar -zxvf dataset.tar.gz
+
+# download test data
+
+cd /incremental_learning/video/
+wget https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/video.tar.gz
+tar -zxvf video.tar.gz
+ 
+```
+
+#### 2.3.3 Scripts prepare
+
+In incremental_learning jobs, the following scripts will be indispensable:
+
+- train.py: script for model fine-tuning/training.
+- eval.py: script for model evaluate.
+- inference.py: script for data inference.
+
+You can also find demos [here](/examples/incremental_learning/helmet_detection).
+
+Some interfaces should be learn in job pipeline:
+
+- `BaseConfig` provides the capability of obtaining the config from env
+
+```python
+
+from sedna.common.config import BaseConfig
+
+train_dataset_url = BaseConfig.train_dataset_url
+model_url = BaseConfig.model_url
+
+```
+
+- `Context` provides the capability of obtaining the context from CRD
+
+```python
+from sedna.common.config import Context
+
+obj_threshold = Context.get_parameters("obj_threshold")
+nms_threshold = Context.get_parameters("nms_threshold")
+input_shape = Context.get_parameters("input_shape")
+epochs = Context.get_parameters('epochs')
+batch_size = Context.get_parameters('batch_size')
+
+```
+
+- `datasources` base class, as that core feature of sedna require identifying the features and labels from data input, we specify that the first parameter for train/evaluate of the ML framework
+
+```python
+from sedna.datasources import BaseDataSource
+
+
+train_data = BaseDataSource(data_type="train")
+train_data.x = []
+train_data.y = []
+for item in mnist_ds.create_dict_iterator():
+     train_data.x.append(item["image"].asnumpy())
+     train_data.y.append(item["label"].asnumpy())
+```
+
+-  `sedna.core` contain all edge-cloud features,  Please note that each feature has its own parameters.
```

Comment:
1. Typo. Please -> please
2. "Each feature has its own parameters." It is confusing for a machine-learning developer, taking seconds to learn what is happening here. We do not mean the machine-learning feature of samples, right? Just say the incremental-learning job.

Reply:
`feature ` here means `incremental-learning `, `lifelong-learning`... In different `feature`, the initial parameters is need to pay attention